janusgraph-codepipelines-ci/pipe.json

```diff
@@ -0,0 +1,15 @@
+{
+    "j1pass" : {
+        "bdb" : { "MODULE" : "janusgraph-berkeleyje" },
+        "lucene" : { "MODULE" : "janusgraph-lucene", "ARGS" : "-DthreadCount=1" },
```

Comment:
threadCount=1 originated during some Travis debugging but I don't think it ever actually solved any problems. I'd suggest removing it here and throughout.